code/unclassified/biggest_Of_N_Numbers/biggest_of_n_numbers.cpp

```diff
@@ -0,0 +1,25 @@
+// compile with c++11 standard
+
+#include <iostream>
+#include <vector>
+#include <algorithm>
+using namespace std;
+
+int biggest_of_n(vector<int> v)
```

Comment:
No need for a separate function.

Reply:
@arnavb plz review my new commit [here](https://github.com/OpenGenus/cosmos/pull/2990)